LICENSE

```diff
@@ -0,0 +1,5 @@
+Copyright 2017 Citrix
```

Comment:
If we're adding a file anyway we might as well add the new copyright headers with Cloud Software Group

Reply:
See [the SDK change recently merged into XAPI for an example ](https://github.com/xapi-project/xen-api/commit/79c5ec26d5f639fbb556f9071d242f3e425bc577#diff-af5fdcef76e2a64789240afc2fc623478436450488ee232210a85f1d716515d9)

Reply:
'Copyright (c) Cloud Software Group, Inc.'